packages/l1-confirmation-checker/scripts/utils.ts

```diff
@@ -0,0 +1,67 @@
+
+import { NodeInterface__factory } from "@arbitrum/sdk/dist/lib/abi/factories/NodeInterface__factory"
+import { SequencerInbox__factory } from "@arbitrum/sdk/dist/lib/abi/factories/SequencerInbox__factory"
+import { NODE_INTERFACE_ADDRESS } from "@arbitrum/sdk/dist/lib/dataEntities/constants"
+import { getL2Network } from "@arbitrum/sdk"
+import { providers, BigNumber } from "ethers";
+
+export const checkConfirmation = async (txHash: string,  l2Provider:providers.JsonRpcProvider) => {
```

Comment:
maybe a quick summary of what the function does / returns (let's have it actually return the # of confirmations as well)